README.md

```diff
@@ -79,7 +121,9 @@ You will require the following environment variables in your shell (or in the ca
 
 ## Artifact Storage
 
-**AWS DynamoDB** was used to keep the demo as simple and self-contained as possible, with the view than it would be replaced by an actual artifact store for a real impelmentation. For this reason the demo requires AWS credentials for some actions.
+Artifact Storage actions expect an `aws.env` file to communicate with AWS infrastructure. This feature is an important precursor for the "verify" part of "trust and verify" pipelines so that verification reports can be build ontop of this metadata (something large organizations may desire for a "single path to production").
+
+**AWS DynamoDB** was used to keep the demo as simple and self-contained as possible (ie not have to manager an Artifactory or Nexus instance just for this demo), with the view than it would be replaced by an actual artifact and properties store for a real impelmentation.
```

Comment:
```suggestion
**AWS DynamoDB** was used to keep the demo as simple and self-contained as possible (ie not have to manage an Artifactory or Nexus instance just for this demo), with the view that it would be replaced by an actual artifact and properties store for a real implementation.
```